src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -196,24 +196,34 @@
         <Grid.ColumnDefinitions>
             <ColumnDefinition MaxWidth="500"
                               MinWidth="1"
-                              Name="LibraryViewColumn">
-                <ColumnDefinition.Width>
+                              Name="LeftExtensionsViewColumn">
+                <!--<ColumnDefinition.Width>
                     <Binding Path="LibraryWidth"
                              Mode="TwoWay"
                              Converter="{StaticResource ConsoleHeightConverter}" />
-                </ColumnDefinition.Width>
+                </ColumnDefinition.Width>-->
             </ColumnDefinition>
             <!--MinWidth fix for GridSplitter dragging in frameless window-->
             <ColumnDefinition Width="Auto" />
-            <ColumnDefinition Width="5*" />
+            <ColumnDefinition Width="5*" MinWidth="200" />
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition MaxWidth="500"
+                              MinWidth="1"
+                              Name="RightExtensionsViewColumn">
+                <!--<ColumnDefinition.Width>
+                    <Binding Path="LibraryWidth"
+                             Mode="TwoWay"
+                             Converter="{StaticResource ConsoleHeightConverter}" />
+                </ColumnDefinition.Width>-->
+            </ColumnDefinition>
         </Grid.ColumnDefinitions>
 
         <!--Titlebar-->
         <Border BorderBrush="#FF3F4040"
                 BorderThickness="0,0,0,1"
                 Grid.Row="0"
                 Grid.Column="0"
-                Grid.ColumnSpan="3"
+                Grid.ColumnSpan="5"
```

Comment:
If you guys find it difficult to understand why I made this change, that indicates team need knowledge share with me on WPF UI grid. I have shared why I had to expand the number here with @scottmitchell but would be happy to cover this with team